sdk/tables/azure-data-tables/README.md

```diff
@@ -160,12 +140,16 @@ entity = {
     'price': '5'
 }
 ```
-* **Create** - Adds an entity to the table.
-* **Delete** - Deletes an entity from the table.
-* **Update** - Updates an entities information by either merging or replacing the existing entity.
-* **Query** - Queries existing entities in a table based off of the QueryOptions (OData).
-* **Get** - Gets a specific entity from a table by partition and row key.
-* **Upsert** - Merges or replaces an entity in a table, or if the entity does not exist, inserts the entity.
+* **Create** - Add an entity to the table.
+* **Delete** - Delete an entity from the table.
+* **Update** - Update an entity's information by either merging or replacing the existing entity.
+    * `UpdateMode.MERGE` will add new properties to an existing entity it will not delete an existing properties
+    * `UpdateMode.REPLACE` will replace the existing entity with the given one, deleting any existing properties not included in the submitted entity
+* **Query** - Query existing entities in a table based off of the QueryOptions (OData).
```

Comment:
What's `QueryOptions`?

Reply:
`QueryOptions` is the actual object. I'm going to re-word this one.